protos/flyteidl/core/execution.proto

```diff
@@ -18,6 +18,7 @@ message WorkflowExecution {
         FAILED = 6;
         ABORTED = 7;
         TIMED_OUT = 8;
+        LAUNCHED = 9;
```

Comment:
Should we added a comment about what this means since it's not immediately clear how this differs from QUEUED?

Reply:
Actually 'QUEUED' might be a better name. cc @kumare3